contracts/Ticket.sol

```diff
@@ -65,6 +67,17 @@ contract Ticket {
         return newTicketId;
     }
 
+    /**
+     * transfers ticket to the specified address
+     *
+     * param ticketID ID of ticket to transfer
+     * param transferTo address to transfer ticket to
+     */
+    function transferTicket(uint256 ticketId, address transferTo) public validTicketId(ticketId) ownerOnly(ticketId) {
```

Comment:
Perhaps need to implement the ownerOnly() modifier.

Reply:
Maybe there is a need for a more restrictive modifier? Since we only want ticket to be transferable between contract and owner only (no owner to owner)